extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -888,10 +888,15 @@ String generateSequenceName(
   @VisibleForTesting
   String generateSequenceName(int groupId)
   {
+    TaskGroup taskGroup = taskGroups.get(groupId);
+    if (taskGroup == null) {
+      log.warn("group id: %s not found in taskGroups, cannot generate sequence number", groupId);
```

Comment:
The log level matters. This should be ERROR or FATAL (well, we have `Logger.wtf() method instead of FATAL).

Reply:
yeah, this should be error. will edit.